turbo.json

```diff
@@ -0,0 +1,27 @@
+{
+  "tasks": {
+    "@storyblok/field-plugin#build": {
+      "outputs": ["dist/**"],
+      "dependsOn": ["@storyblok/manifest-helper#build"]
```

Comment:
Hey @eunjae-lee 🙌 

I'm wondering if the `lib-helpers` should be also built so the helpers get copied to the `field-plugin/dist` folder. 🤔 
Or would this routine be triggered at another time?

Reply:
Hey @demetriusfeijoo good point!

![Screenshot 2024-09-25 at 16 14 26](https://github.com/user-attachments/assets/ace0a403-7db1-4eaa-a165-710bfdfefb02)

After `@storyblok/manifest-helper` and `@storyblok/field-plugin` are built, the `lib-helpers` get built, and its outputs are copied to `field-plugin/dist/` folder.

Reply:
Oh, now I got it correctly, @eunjae-lee.
Thanks 🙏